src/cli.cpp

```diff
@@ -241,7 +241,7 @@ void CLI::printID(DWORD *pid) {
     int i, digit = 0;
 
     // Convert PID to ascii-number (=raw)
-    sprintf(raw, "%09u", pid[0]);
+    snprintf(raw, sizeof(raw), "%09u", pid[0]);
```

Comment:
ah yes, we should've be using snprintf here

perhaps when we switch from fmt to pocoproject this will be a good candidate to abstract